ansible-tests/validations/instackenv.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: undercloud
+  become: false
+  vars:
+    metadata:
```

Comment:
Add this validation to the 'hardware-discovery' group so that mistral based workflow can classify it.